apps/backend/src/prisma/prisma.service.ts

```diff
@@ -12,4 +12,22 @@ export class PrismaService extends PrismaClient implements OnModuleInit {
       await app.close();
     });
   }
+
+  async clean() {
+    const tablenames = await this.$queryRaw<
+      Array<{ tablename: string }>
+    >`SELECT tablename FROM pg_tables WHERE schemaname='public'`;
+
+    const tables = tablenames
+      .map(({ tablename }) => tablename)
+      .filter((name) => name !== '_prisma_migrations')
+      .map((name) => `"public"."${name}"`)
+      .join(', ');
+
+    try {
+      await this.$executeRawUnsafe(`TRUNCATE TABLE ${tables} CASCADE;`);
+    } catch (error) {
+      console.log({ error });
+    }
+  }
```

Comment:
personally, I think use separate database may be more practical. This code is seem very destructive if miss use.

Reply:
I think we may start test db from docker compose and run it with inject env to prisma or sth like that.

Reply:
also to cleanup, may delete the db with remove of docker volume